src/libstd/fs.rs

```diff
@@ -21,7 +21,9 @@ use crate::time::SystemTime;
 /// it was opened with. Files also implement [`Seek`] to alter the logical cursor
 /// that the file contains internally.
 ///
-/// Files are automatically closed when they go out of scope.
+/// Files are automatically closed when they go out of scope.  Errors detected
+/// on closing are ignored by the implementation of `Drop`.  Use the method
+/// `sync_all` if these errors must be manually handled.
```

Comment:
It may be beneficial to clarify how sync_all is relevant to detecting close errors. Sync_all does not close the file so there is no obvious connection for someone not already familiar with failure modes of close. What do you think?

The way I would prefer to see this is by keeping the text written here but turning sync_all into a link to a new subsection of the sync_all method documentation that explains about how it is useful for this purpose.

Reply:
I thought about what to write in `sync_all` but am pulling a blank.  Thoughts?

Reply:
The wording added at the top of File is:

> Use the method [`sync_all`](#) if these errors must be manually handled.

So I would expect some explanation under `sync_all` of how to use `sync_all` to manually handle those sorts of errors that this sentence refers to.

You could explain some ways that closing a file can fail and how code can be written using `sync_all` to handle those failures correctly.